.github/workflows/psql-release-notifier.yaml

```diff
@@ -0,0 +1,57 @@
+# #############################################################################
+# Copyright (c) 2023 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# #############################################################################
+
+name: PostgreSQL new release notifier
+
+on:
+    workflow_dispatch:
+    schedule:
+        # Run every day at 01:00 at night
+        - cron: "0 1 * * *" # UTC time
+env:
+    TRACTUSX_MAILINGLIST: _CHANGE_ME_@MAIL.COM 
+    DEVSECOPS_NOTIFICATION_EMAIL: eclipse.tractusx@gmail.com
```

Comment:
Who is owning this Email address? Is there documentation on how to maintain this?
"@gmail" seems weird to me. Isn't there a "@eclipse.org" address, that we could use?

Reply:
Right, that was the goto advice I got from @Siegfriedk and Frederic https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/3702 requesting originally a Jenkins job. Not sure if we can have @eclipse.org, I know we have one @catena-x.net but don't think it's relevant for tractus-x usage. 

Mail details are stored in Vault at devsecops/mail/ path.

Reply:
Ok fine with that. 
The only risk I see is the details being stored in Vault, which is consortia only and has a TTL. 
Moving forward, we need good documentation. Right now the recovery email and EF contacts are worth mentioning from my point of view